### PR TITLE
Change syntax for colors to adapt to use of CommonMark

### PR DESCRIPTION
**Pull request by Martchus**
This should only be merged if the upcoming Markdown change in openQA is merged and deployed.

### CODE REVIEW

File: openqa_review/openqa_review.py [hunk 1]
```diff
@@ -195,9 +195,9 @@ def pluralize(_1, _2, plural):
 """)
 
 status_badge_str = {
-    'GREEN': '<span style="color: green;">Green</span>',
-    'AMBER': '<span style="color: #FFBF00;">Amber</span>',
-    'RED': '<span style="color: red;">Red</span>',
+    'GREEN': '{{color:green|Green}}',
```
Comment (kraih) [inline openqa_review/openqa_review.py]: Color names don't work, just hex values.

Comment (Martchus) [reply openqa_review/openqa_review.py]: Fixed

**Comment by Martchus** [comment]
You know the codebase of these scripts best so what would be the best solution? Having two different modes for the generator? Handling this syntax in the review script as well when converting to HTML? Do the conversion from `span` tags to the new syntax in a post-processing step if required?

**Comment by okurz** [comment]
After thinking about it and checking my emails I did not see anyone referencing anymore reports on openQA itself except Weihua for Security. For our current SLE product in development, SLE12SP5, I also only found https://openqa.suse.de/group_overview/167#comment-192096 (same team) which looks like a customized openqa-review report, all other teams either do not use comments on openQA anymore or use a more simplified markup without color so with this I guess we can live with just keep it as is and not do anything at all. It is good to have this PR for reference though. If you are ok with this just close the PR.

**Comment by Martchus** [comment]
ok

**Pull request closed without merging**